src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -93,6 +93,8 @@ internal override void Initialize(ApplicationEngine engine)
         protected override void PostPersist(ApplicationEngine engine)
         {
             base.PostPersist(engine);
+            // PersistingBlock is null when running under the debugger
+            if (engine.Snapshot.PersistingBlock == null) return;
```

Comment:
If it is not executed in a block, the `PostPersist()` won't be triggered.

Reply:
So the changes should be reverted.

Reply:
Done